src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/RequestExtensionCallbackHandlerTest.java

```diff
@@ -161,7 +164,8 @@ void shouldReturnExpectedResponse_whenAlreadyAgreed() {
             CallbackParams params = callbackParamsOf(
                 of(PROPOSED_DEADLINE, proposedDeadline,
                    EXTENSION_ALREADY_AGREED, "Yes",
-                   RESPONSE_DEADLINE, responseDeadline
+                   RESPONSE_DEADLINE, responseDeadline,
+                   LEGACY_CASE_REFERENCE, REFERENCE_NUMBER
```

Comment:
not sure if super important, but if you accidentally change one of these field's values in the CallbackHandler class, tests would not pick it up and you wouldn't know about it until e2e

Reply:
I will pass this. I hear you what you are trying to say. Would like to stay consistent with the approach for other constants used before in the same test class.